### PR TITLE
Update to 1.0.0-alpha.9

### PR DESCRIPTION
**Pull request by cyborgthefirst**
I've updated the project to use the latest version of grpc-swift. I also added the option to generate the code using a docker image I'm hosting and maintaining so that people do not have to install `protoc` on their machines.

### CODE REVIEW

**Comment by vyshane** [comment]
Thanks for the PR @cyborgthefirst. I will get to it within the next 24 hours.

File: Makefile [hunk 1]
```diff
@@ -7,3 +10,11 @@ protobuf:
...
 	protoc test_scenarios.proto --swift_out=../CombineGRPCTests/Generated/; \
 	protoc test_scenarios.proto --grpc-swift_out=../CombineGRPCTests/Generated/
+
+protobuf_docker:
+	mkdir -p Tests/CombineGRPCTests/Generated
+
+	docker run --rm -v "${PWD}":/protobuf -w="/protobuf" cyborgthefirst/grpc-swift:1.0.0-alpha.9 \
+	protoc ${PROTO_DIR}/*.proto --swift_opt=FileNaming=DropPath,Visibility=Public \
+	--swift_out=${OUT_DIR} \
+	--grpc-swift_out=FileNaming=DropPath,Visibility=Public,Server=true,Client=true:${OUT_DIR}
```
Comment (vyshane) [inline Makefile]: Should we omit explicitly listing these out if they are the default options?

```
FileNaming=DropPath,Visibility=Public,Server=true,Client=true
```

File: Makefile [hunk 2]
```diff
@@ -7,3 +10,11 @@ protobuf:
 	cd Tests/Protobuf; \
```
Comment (vyshane) [inline Makefile]: Can you update the `protobuf:` task to use `PROTO_DIR` & friend, similar to your new addition that uses Docker container to generate code from the protos?

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,6 @@
+PROTO_DIR = Tests/Protobuf
+OUT_DIR = Tests/CombineGRPCTests/Generated
```
Comment (vyshane) [inline Makefile]: Perhaps this, to be explicit?

```suggestion
PROTO_GEN_DIR = Tests/CombineGRPCTests/Generated
```

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    && unzip protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr \
+    && rm -rf protoc-${PROTOC_VERSION}-linux-x86_64.zip
+
+RUN git clone -b ${GRPC_SWIFT_VERSION} https://github.com/grpc/grpc-swift \
+    && cd grpc-swift \
+    && make plugins \
+    && find . -type f -name "protoc-gen-swift" -exec cp {} /usr/bin \; \
+    && find . -type f -name "protoc-gen-grpc-swift" -exec cp {} /usr/bin \; \
+    && cd / \
+    && rm -rf grpc-swift
```
Comment (vyshane) [inline Dockerfile]: Nice 👍

File: Makefile [hunk 5]
```diff
@@ -7,3 +10,11 @@ protobuf:
 	cd Tests/Protobuf; \
 	protoc test_scenarios.proto --swift_out=../CombineGRPCTests/Generated/; \
 	protoc test_scenarios.proto --grpc-swift_out=../CombineGRPCTests/Generated/
+
+protobuf_docker:
+	mkdir -p Tests/CombineGRPCTests/Generated
+
+	docker run --rm -v "${PWD}":/protobuf -w="/protobuf" cyborgthefirst/grpc-swift:1.0.0-alpha.9 \
```
Comment (vyshane) [inline Makefile]: Will you be publishing updated images as the gRPC Swift project publish new releases?

File: Makefile (hunk 1 above)
Comment (cyborgthefirst) [reply Makefile]: Yes definitely! Did not realise that I added the default parameters when testing.

File: Makefile (hunk 5 above)
Comment (cyborgthefirst) [reply Makefile]: Yes, I use gRPC-swift in another project at work, where I will need to update these images anyway.

File: Dockerfile (hunk 4 above)
Comment (cyborgthefirst) [reply Dockerfile]: Thanks. I will open a PR in grpc-swift as well, because as far as I know this is the only working 1.0.0-alpha.9 Dockerfile so far.

**Comment by vyshane** [comment]
Merged, thanks for your contribution :)

**Pull request merged**